integration/rules/javascript_test.go

```diff
@@ -7,56 +7,26 @@ func TestJavascriptLangLoggerSummary(t *testing.T) {
 	runRulesTest("javascript/lang/logger", "summary", "javascript_lang_logger", t)
 }
 
-func TestJavascriptLangLoggerDataflow(t *testing.T) {
-	t.Parallel()
-	runRulesTest("javascript/lang/logger", "dataflow", "javascript_lang_logger", t)
-}
-
 func TestJavascriptLangSessionSummary(t *testing.T) {
 	t.Parallel()
 	runRulesTest("javascript/lang/session", "summary", "javascript_session", t)
```

Comment:
Can we remove the `"summary"` argument now?

Reply:
yeah definitely, was hoping to remove rule name too, 
however idea was to do it in another pass once tests are refractored.